cmd/get/volume.go

```diff
@@ -285,9 +348,12 @@ $ storageos get volume --namespace my-namespace-name my-volume-name
 		SilenceUsage: true,
 	}
 
-	cobraCommand.Flags().BoolVarP(&c.allNamespaces, "all-namespaces", "A", false, "retrieves volumes from all accessible namespaces. This option overrides the namespace configuration")
-
+	cobraCommand.Flags().BoolVarP(&c.allNamespaces, "all-namespaces", "A", false, "retrieves volumes from all namespaces. This option overrides the namespace configuration")
 	flagutil.SupportSelectors(cobraCommand.Flags(), &c.selectors)
 
+	cobraCommand.Flags().StringVarP(&replicaHealthState, "replica-health-state", "R", "", "shows only volumes with replicas in this state. Valid options are \"recovering\", \"provisioning\", \"provisioned\", \"syncing\", \"ready\", \"deleted\" and \"failed\"")
+	cobraCommand.Flags().StringVarP(&masterHealthState, "master-health-state", "M", "", "shows only volumes in this state. Valid options are \"online\" and \"offline\"")
+	cobraCommand.Flags().IntVarP(&replicaCount, "replica-count", "r", -1, "show only volume with this many replicas. Based on volume configuration. Valid number in the interval [0,5]")
+
```

Comment:
This would be a it easy to read if we just stuck the whole string in backquotes \`, then you don't need to escape every speech mark